app/gui/src/project-view/components/CircularMenu.vue

```diff
@@ -139,107 +187,53 @@ function readableBinding(binding: keyof (typeof graphBindings)['bindings']) {
     width: 100%;
     height: 100%;
     pointer-events: all;
-    clip-path: var(--full-ring-path);
+    top: 36px;
+    transition: all ease 0.1s;
+    clip-path: path(
+      'M0,16 V16 A52,52,0,0,0,52,68 A16,16,0,0,0,52,36 A20,20,0,0,1,32,16 A16,16,0,0,0,0,16'
+    );
   }
-
-  &.partial {
-    &:before {
-      top: 36px;
-      clip-path: path(
-        'm0 16a52 52 0 0 0 52 52a16 16 0 0 0 0 -32a20 20 0 0 1-20-20a16 16 0 0 0-32 0'
-      );
-    }
+  &.openedDropdown:before {
+    background: var(--dropdown-opened-background);
+    backdrop-filter: var(--dropdown-opened-backdrop-filter);
+    clip-path: path(
+      'M0,16 V68 A52,52,0,0,0,52,68 A16,16,0,0,0,52,36 A20,20,0,0,1,32,16 A16,16,0,0,0,0,16'
+    );
   }
 }
 
-.More {
-  position: absolute;
-  left: -5.5px;
-  top: 27px;
-  width: 42px;
-  height: 17px;
+:deep(.ColorRing .gradient) {
```

Comment:
Can we just move this style to ColorRing.vue instead of :deep?

Reply:
Done